docs/getting-started/migrating.md

```diff
@@ -0,0 +1,86 @@
+# Migrating from 0.X
+
+!!! tip
+    This page is up-to-date with version `1.0.0-beta.5` of bdk.
+
+So you're ready to migrate to BDK version 1.0, congratulations!
+This document contains some helpful tips that, with the help of some automation, should make the process as seamless as possible.
+
+The below steps are for migrating wallet details from the old [`bdk` v0.30][0] to the new [`bdk_wallet` v1.0][1].
+This procedure can be applied to wallets backed by a SQLite database.
+Of particular concern is the ability to restore the _last known address index_ for each keychain.
```

Comment:
This is essentially the only thing we're doing correct? I would personally say something like 
"to migrate your wallet data to a new version of bdk, all you essentially need to do is grab the [last used addresses] from the old db, add them to the new db and sync to refetch the rest of the data. Note that we don't need to perform a full scan because we will already have the last used addresses"
To make it extra clear